bosch_thermostat_client/db/nsc_icom_gateway.json

```diff
@@ -0,0 +1,292 @@
+                    "id": "roomtemperature",
+                    "name": "Room temperature",
+                    "type": "regular"
+                },
+                "operation_mode": {
+                    "id": "operationMode",
+                    "name": "Operation mode",
+                    "type": "operation_mode"
+                },
+                "setpoint1": {
+                    "id": "manualRoomSetpoint",
+                    "name": "Manual room setpoint",
+                    "type": "setpoint"
+                },
+                "setpoint2": {
+                    "id": "temporaryRoomSetpoint",
+                    "name": "Temporary room setpoint",
+                    "type": "setpoint"
+                },
+                "status": {
+                    "id": "status",
+                    "name": "Heating circuit status",
+                    "type": "regular"
+                },
+                "activeProgram": {
+                    "id": "activeSwitchProgram",
+                    "name": "Active Switch Program",
+                    "type": "activeProgram"
+                }
+            },
+            "sensors": {
+                "supplyTemperatureSetpoint": {
+                    "id": "supplyTemperatureSetpoint",
+                    "name": "supplyTemperatureSetpoint"
+                },
+                "suWiThreshold": {
+                    "id": "suWiThreshold",
+                    "name": "Summer Winter Threshold"
+                },
+                "switchProgramMode":{
+                    "id": "switchProgramMode",
+                    "name": "switchProgramMode"
+                },
+                "pumpModulation": {
+                    "id": "pumpModulation",
+                    "name": "Pump Modulation"
+                },
+                "suWiSwitchMode": {
+                    "id": "suWiSwitchMode",
+                    "name": "Summer Winter Switchmode"
+                },
+                "currentSuWiMode": {
+                    "id": "currentSuWiMode",
+                    "name": "Current Summer Winter Mode"
+                }
+            }
+        },
+        "dhwCircuits": {
+            "hastates": [
+                {
+                    "haname": "off",
+                    "boschname": [
+                        "Off"
+                    ]
+                },
+                {
+                    "haname": "on",
+                    "boschname": [
+                        "on", "low"
+                    ]
+                },
+                {
+                    "haname": "eco",
+                    "boschname": [
+                        "eco"
+                    ]
+                },
+                {
+                    "haname": "eletric",
+                    "boschname": []
+                },
+                {
+                    "haname": "performance",
+                    "boschname": [
+                        "ownprogram"
```

Comment:
delete this.

Reply:
The whole `"haname": "performance",` or just the `ownprogram` from `boschname`?